crates/e2e/src/nodes/mod.rs

```diff
@@ -43,6 +43,8 @@ impl Node {
             "1",
             "--timestamp",
             "1577836800",
+            "--hardfork",
+            "cancun",
```

Comment:
Unless CI also starts failing if we don't set this value here I wouldn't do it.
Just one more thing to remember when the next hardfork happens.

Reply:
Oh sorry forgot to delete this.